README.md

```diff
@@ -1,6 +1,7 @@
 # awesome-nlp 
 
 [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+[![Build Status](https://travis-ci.org/keon/awesome-nlp.svg?branch=master)](https://travis-ci.org/keon/awesome-nlp)
```

Comment:
Hey @keon , let's wait till we have a green build, shall we? What do you think?

Reply:
@NirantK sure. It is not urgent :)